tests/test_benchmark_modules/test_hf.py

```diff
@@ -33,3 +36,55 @@ def test_get_torch_dtype(test_device, torch_dtype_is_set, bf16_available, expect
         )
         == expected
     )
+
+def test_safetensors_check():
+    """Test the safetensors availability check functionality."""
+    
+    # Mock benchmark config with safetensors check enabled
+    benchmark_config = MagicMock(spec=BenchmarkConfig)
+    benchmark_config.only_allow_safetensors = True
+    benchmark_config.cache_dir = ".cache"
+    benchmark_config.api_key = None
+    benchmark_config.trust_remote_code = False
+    benchmark_config.run_with_cli = True
```

Comment:
This test seems to give false confidence, since currently the `BenchmarkConfig` does not have an `only_allow_safetensors` attribute. 

There is already a `benchmark_config` fixture, so you could just use that as follows:

```suggestion
def test_safetensors_check(benchmark_config):
    """Test the safetensors availability check functionality."""
    benchmark_config.only_allow_safetensors = True
```